torch/nn/modules/module.py

```diff
@@ -531,6 +533,35 @@ def get_buffer(self, target: str) -> "Tensor":
 
         return buffer
 
+    def get_extra_state(self) -> Dict[str, Any]:
```

Comment:
Does this really have to return a Dict? Why not Any?

Reply:
Changed to Any